.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Global rule:
+*     @xmtp/engineering
```

Comment:
I figure since this spans a lot of engineers it might be okay to use engineering here. But also could be convinced to use a smaller subset of proto pod people if we want to create a team?

Reply:
Seems good for now, given its unclear exactly who should be on that list. Once it becomes clear the list can be tightened down